app/components/zig_codeblocks.py

```diff
@@ -146,173 +97,58 @@ async def _prepare_reply(
         )
 
     codeblocks = list(extract_codeblocks(message.content))
-    file_highlight_note = 'On desktop, click "View whole file" to see the highlighting.'
-    if codeblocks and any(block.lang == "zig" for block in codeblocks):
-        zig_code = process_discord_markdown(message.content, only_code=True)
-        zig_code += f"\n{file_highlight_note}" if attachments else ""
-        return _split_codeblocks(zig_code), attachments
+    if (
+        codeblocks
+        and len(message.content) <= 2000
+        and any(block.lang == "zig" for block in codeblocks)
+    ):
```

Comment:
This loop runs for code blocks that aren't Zig code blocks, turning something like
````md
```zig
foo bar baz
``````rs
foo rar raz
```
````
into
````md
```ansi
foo bar baz```

```ansi
foo rar raz```
````


```suggestion
    codeblocks = [c for c in extract_codeblocks(message.content) if c.lang == "zig"]
    if codeblocks and len(message.content) <= 2000:
```

Reply:
I have no clue why that suggestion includes the removed lines too; for clarity, this is what the diff looks like:
```diff
-    codeblocks = list(extract_codeblocks(message.content))
+    codeblocks = [c for c in extract_codeblocks(message.content) if c.lang == "zig"]
-    if (
-        codeblocks
-        and len(message.content) <= 2000
-        and any(block.lang == "zig" for block in codeblocks)
-    ):
+    if codeblocks and len(message.content) <= 2000:
```